server/public/model/utils.go

```diff
@@ -336,6 +336,11 @@ func (er *AppError) Wrap(err error) *AppError {
 	return er
 }
 
+func (er *AppError) WipeDetailed() {
+	er.wrapped = nil
+	er.DetailedError = ""
```

Comment:
One last suggested change would be to make `DetailedError` to be unexported. So that there's no chance of this mistake happening again in future. But not blocking.

Reply:
It actually needs to be exported to be at all accessible during development or when `EnableDeveloper` is on, so I don't think we can make it private. That'd also technically be a breaking change as well

Reply:
>It actually needs to be exported to be at all accessible during development or when EnableDeveloper is on,

Oh I might have missed cases where that string is explicitly set to something else. Yeah, in that case, we would need to do it via a method.

> That'd also technically be a breaking change as well

Not anymore, because we have moved the public module under 0.1x series. So anything can be done.

Reply:
If it's all the same to you @agnivade, I would prefer to have it exported. Sound good?

Reply:
Sure